Classification/SGD/src/main/java/org/tribuo/classification/sgd/crf/SeqTest.java

```diff
@@ -118,11 +114,6 @@ public String getOptionsDescription() {
          */
         @Option(longName = "model-hashing-salt", usage = "Salt for hashing the model.")
         public String modelHashingSalt = "";
-        /**
-         * Load in the model in protobuf format.
-         */
-        @Option(longName = "protobuf-model", usage = "Load the model from a protobuf. Optional")
```

Comment:
Again isn't this a breaking change? Code that was using this arg will now crash, even though it was doing things the 'right' way